_episodes/interviews/2022-12-09-seaton-ullberg.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Kernel Density Estimation with Seaton Ullberg"
+date: 2022-12-02T17:04:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2022-12-06-seaton-ullberg.mp3
+duration: "43:14"
+length: "41503347"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Seaton Ullberg](https://github.com/seatonullberg), developer of [Kernel Density Estimation](https://github.com/seatonullberg/kernel-density-estimation), a library that helps to calculate the kernel desnity estimation using data points.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+- [@0:00] - Introduction
+- [@0:55] - Understanding Kernel Density Estimation library and how it works
+- [@17:49] - Kernel Density Estimation vs other competing algorithms
+- [@24:05] - Application uses of Kernel Density Estimation
+- [@33:38] - Why write this library
+- [@35:03] - Why use Rust when re-writing this library
+- [@40:26] - Seaton using Float64 as a feature
```

Comment:
What is Float64 here? Should it link to something? Or is it `f64` (in which case we should use that)?